Main.java

```diff
@@ -667,10 +679,15 @@ private static void finishFight(Client winner, Client loser) {
       winner.giveItem(Game.Item.HPOTION);
       msg(winner, "You found 1 healing potion!");
     } else {
-      int rnd = Utils.rndInRange(1,4);
+      // logic for looting bots is here
```

Comment:
It's not only bots, right? You can loot other players as well.

Reply:
As of now if you win human player you're always given a health potion (line 678-680). Do you mean we should change that and allow human players to be looted? Should the item disappear in this case from the loser's inventory? If that's the case, it might be too harsh on the loser:
- they lost a fight
- they have to wait for health to restore
- they gained nothing from the fight (no items, no experience)
- they also lost an item from their inventory?!

Reply:
@Riboloff has confirmed that we shouldn't be able to loot other players yet.